Beyond09-SOW.md

```diff
@@ -0,0 +1,508 @@
+
+**Demo contents**
+
+Your demo presentation should include the following elements:
+
+1. One or more video segments showing how the features you've implemented look
+   and work from a user's point of view.
+2. An explanation of how you've tested the features you've developed. You should
+   include the following details:
+
+    1. Which manual tests were carried out (If any). How does one decide if a 
+       test was successful?
+    2. Which automated tests were written
+    3. Are failure scenarios being tested? In which way?
+    4. What aspects of the features you've written are not being tested?
+3. A demonstration of the application tests being run in GitHub actions.
+
+### Demo 4 (June 11th): Final Project Demo
+
+The purpose of this demo is to present your entire development process and 
+should include:
+
+1. Project background - Explain in general the motivation and the requirements.
+2. Technical ideas and architecture - Explain test high-level technical 
+   ideas behind your project, please try to present concepts graphically, 
+   possibly by using ERD and DFD charts.
+4. Demo - A video that presents your full system (UI, DB, APIs, CI)
+5. Development process - share the process you made
+6. Development challenges
+7. What did you learn
+
+## Division into project groups (till March 19th)
+
+After the first 2 weeks, you'll be divided into teams - each team with 4-6 students.
+
+Each team will have the following mentors:
+
+* Team 1: Yftach Herzog, Yuval Vinokur, Nitzan Mordechai, Sebastian Scheinkman
+* Team 2: Kobi Hakimi, Moshe Sheena, Or Mergi, Ygal Blum
+* Team 3: Yariv Rachmani, Benny Rochwerger, Amos Mastbaum, Daniel Keler
+* Team 4: Nati Friedman, Omer Amsalem, Noam Finkelstein, Alicia Petrunin, Galit Rosenthal
+
+## Project ideas
+
+You can consider below ideas or think up your own.
+
+* Dating website
+* Drop shipping web store - selling other people's products on your website
+* Personal password maintenance application
+* Course certificate generation and distribution platform
+* An online riddle or math game
+
+Please avoid the following ideas as they have already been implemented too many
+time in the past by Beyond students:
+
+* Platform for finding part-time jobs for students.
+* Platform for academic courses and grading.
+* Platform for finding/recruiting jobs/employess.
+* Platform for for finding roomates or appartments.
+* Platform for freelancers to offer their services.
+* P2P Lending service
+
+__Note:__ The project used as a learning platform, it should not be
+ unique or amazing, it could defeat learning process and goals.
+
+## Syllabus (list of sessions)
+
+| #    | Date                | Topic(s)                           | Lecturer        | Location        |
+|------|---------------------|------------------------------------|-----------------|-----------------|
+|-     |Wed, Feb 19, 2023    |First challenge hand-off and student selection (30m pre-course session)|Barak Korren|Online/Remote|
```

Comment:
What does it mean student selection in the context of this meeting. Are students been selected there?

Reply:
We talk about the selection process